app.rb

```diff
@@ -0,0 +1,132 @@
+require './book'
+require './persons'
+require './rental'
+require './student'
+require './teacher'
```

Comment:
You seem to have imported `student` and `teacher` whereas the files are named `students` and `teachers`. Kindly reconcile the file names.

![image](https://user-images.githubusercontent.com/20554583/216769143-b234040d-b86e-4239-9c2e-247ff5434879.png)